create_user.py

```diff
@@ -0,0 +1,17 @@
+#! /usr/bin/env python3
+
+import getpass
+import json
+
+from werkzeug.security import generate_password_hash
+
+with open('users.json') as u:
+    users = json.load(u)
+
+user = input('username> ')
+password = getpass.getpass(prompt='password> ')
+
+users[user] = generate_password_hash(password)
```

Comment:
i really appreciate the practicality of this; it's effective without being overengineered.

Reply:
I’m glad you appreciate this. I really didn’t want an endpoint to handle this. Only the user running the app or one with permission to.